setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='stylize-datasets',
+    version='1.0',
+    packages=[''],
+    scripts=['models/download_models.sh'],
```

Comment:
please remove this line, as this script doesn't exist anymore =)

Reply:
@Hvitgar - seems through we should have something like a `download_models.sh`(again); in DLC we do so this way (and this has bee robust for us): https://github.com/DeepLabCut/DeepLabCut/blob/master/deeplabcut/pose_estimation_tensorflow/models/pretrained/pretrained_model_urls.yaml

Reply:
@MMathisLab sure, that would be nice, but we referred to a google drive url before, which was not under our control, resulting in broken links (see #14). Apart from re-hosting model weights from other people, I don't see a way to provide a stable working download script.

Reply:
I would argue for a re-hosting :). Happy to talk "offline" if useful too!

Reply:
@MMathisLab as I am not part of Bethgelab anymore, this would require someone to maintain it on their side. On top, we would need to make sure we don't violate licenses by re-hosting existing model weights